CHANGELOG.md

```diff
@@ -5,6 +5,36 @@ This changelog is according to [Keep a Changelog](http://keepachangelog.com).
 All notable changes to this project will be documented in this file.
 We will follow [Semantic Versioning](http://semver.org/) from version 2 and onwards.
 
+## 1.4: July 19th, 2016
+
+### Changed
+    * Improves feedback texts of assessments.
+    * Improves Russian transliteraion.
+    * Determine length of title based on pixels in stead of on number of characters.
```

Comment:
Maybe try: `based on the width in pixels instead of on the number of characters.`